vorc_gazebo/launch/gymkhana.launch

```diff
@@ -8,11 +8,11 @@
   <!-- If true, run gazebo in verbose mode -->
   <arg name="verbose" default="false"/>
   <!-- Set various other gazebo arguments-->
-  <arg name="extra_gazebo_args" default=""/>
+  <arg name="extra_gazebo_args" default="--verbose"/>
   <!-- Start in a default namespace -->
   <arg name="namespace" default="cora"/>
   <!-- Do we lock the vessel to the world? -->
-  <arg name="robot_locked" default="false" />
+  <arg name="robot_locked" default="true" />
```

Comment:
Should this be `false`?

Reply:
There's an error from station-keeping plugin about unable to unlock the 2 joints, if it's false. I looked at station-keeping and navigation launch files in VRX, and they both locked it, so I just changed it to true.